### DISCUSSION TITLE
reverse hidden mod?

### DISCUSSION THREAD

**Discussion by Nepp0** [ppy/osu#19044] [Ideas]
circles fade in and you'd have to hit them when they're fully visible

**Comment by nagi-desuuu** [comment]
So what you're proposing is the Fade In osu!mania mod be ported to regular
osu, right?

**Reply by Nepp0** [reply]
oh i didnt know that existed! yeah basically like that but could also be implemented as a mod customisation feature

**Reply by frenzibyte** [reply]
~~osu!(stable) is feature locked, not sure such mods will be backported to that version.~~ As for customising, can you elaborate what should be customisable in the mod?

By "regular osu!", I suppose you mean the osu! ruleset in lazer?
